### DISCUSSION TITLE
欢迎来到Redis中文注释讨论区！

### DISCUSSION THREAD

**Discussion by ncghost1** [CN-annotation-team/redis7.0-chinese-annotated#11] [Announcements]
## 👋 你好吖！这里是Redis中文讨论区！
我们把讨论区作为大家讨论 Redis 的地方，我们希望你可以：
  * 提出你的问题
  * 分享你的观点
  * 与其他社区成员互动
  * 欢迎他人并保持开放的心态。请记住，这是一个社区，是我们一起建立的💪。
  * 与人交流的时候要注意礼貌，语言要文明，别人才会愿意和你交流哦~🍭🍭

**Comment by ZigHuang** [comment]
README.md的项目进度一栏中，针对已经完成翻译的文件，可以添加跳转的url，方便查看

**Reply by ncghost1** [reply]
我们这个表格已经做得很“胀”了，要是再加一列不知道会成什么样子…
也可以在下面加一行提示可以用github的快捷键来搜索？

**Reply by ZigHuang** [reply]
🤣不是的，改成`markdown`的`[title](url)` 这种格式
例如: [t_hash.c](https://github.com/CN-annotation-team/redis7.0-chinese-annotated/blob/7.0-cn-annotated/src/t_hash.c)

**Reply by ncghost1** [reply]
这个可以👍你有空去提个PR我们再详细讨论
对了，以后新 idea 开个新帖子，而不是在这里回复。

**Comment by huihuang-chen** [comment]
感觉我们写注释有时候可能需要加一些文档来给读者梳理流程，正常直接给源码加注释，如果没看过redis源码的人来看可能连找一些功能的入口都找不到，我现在在梳理redis主流程网络通信源码，但是方法调用是分散在各个文件中的，要是其他人读源码，很难将这些源码连起来看
